src/setup/redisConfiguration.ts

```diff
@@ -19,7 +19,13 @@ export async function checkRedisConnection(url: string): Promise<boolean> {
     await client.connect();
     response = true;
   } catch (error) {
-    console.log(`\nConnection to Redis failed. Please try again.\n`);
+    if (error instanceof Error) {
```

Comment:
I don't think we need it and haven't found any updates in the release notes of TS. Can you post the error here?